ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -230,6 +230,16 @@ objects:
         export CLUSTER_ID=$(uuidgen --random)
         export SSH_PUB_KEY=$(cat "${SSH_PUB_KEY_PATH}")
         export PULL_SECRET=$(cat "${PULL_SECRET_PATH}")
+        # check whether we already have a floating ip created
+        LB_FIP=$(openstack floating ip list --format value | awk -F ' ' 'NR==1 {print $2}')
```

Comment:
This can be written as: `openstack floating ip list -f value -c 'Floating IP Address'`, which I'd say is cleaner and more robust (the default view is not really supposed to be used by scripts and can change).

More importantly though, won't this fail if there's another job running already? There will be a floating IP in that case, but it will be in use so we shouldn't be taking it.

Reply:
Indeed, see my change at https://github.com/shiftstack-dev-tools/shiftstack-ci/pull/6/files

Reply:
Yes it is missing the `--status DOWN` check. Nice catch. I will also fix the awks.